rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, book, person)
+    @date = date
+    @person = person
+    @book = book
+
+    # associations
+    book.rentals << self
+    person.rental << self
```

Comment:
- [ ] I think this should be `person.rentals` (notice the 's'), based on what we have in the Person class.